00-HTML-CSS-basics/index_2.3.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta content="name=value,name=value" name="viewport" />
+    <meta charset="utf-8">
+    <link href="style_2.3.css" rel="stylesheet" type="text/css">
+
+ <!-- .......ITEM 6........ 
+    <style type="text/css">
+        
+        * {
+            font-size: 80px;
+        }
+
+    </style> -->
+
+    <title>CSS</title>
+</head>
+<body>
+
+<div class="wrapper">
+    <header class="header">
+        <font>header1</font>
```

Comment:
I don't understand what you mean. I did use font for the item 6 to work, otherwise it wouldn't change the font-size, but I don't understand what you mean with the asterisk thing. Should I delete the fonts? Or replace them for other tags?